Anlab.Mvc/Client/components/OrderForm.tsx

```diff
@@ -57,6 +61,9 @@ export default class OrderForm extends React.Component<undefined, IOrderState> {
             isErrorActive: false,
             errorMessage: '',
             isAdmin: false,
+            status: '',
+            labComments: '',
+            adjustmentAmount: 0,
         };

```

Comment:
@jSylvestre this initial state is starting to get really long.  Something to keep an eye on because it's only going to grow.  Maybe we can do more like what we are doing with payment and start nesting a bit.